.travis.yml

```diff
@@ -33,11 +33,11 @@ env:
     # - DBUILD_ARGS="--cache-from ${DEV_REGISTRY}/builder:latest"
 
     # Set from the repository settings:
-    # - DOCKER_DEV_REGISTRY=quay.io/datawire-dev (or something different if not datawire/ambassador.git)
+    # - DOCKER_DEV_REGISTRY -- e.g. datawire-dev (or something different if not datawire/ambassador.git)
```

Comment:
I like to use strings like `docker.io/datawire-dev`, just to visually root it; so you can understand what that string is.

Reply:
I like that idea -- will do after we get the live stuff updated.